room_assistance/indico_room_assistance/util.py

```diff
@@ -13,3 +13,14 @@ def is_room_assistance_support(user):
     if user.is_admin:
         return True
     return RoomAssistancePlugin.settings.acls.contains_user('room_assistance_support', user)
+
+
+def event_has_room_with_support_attached(event):
+    from indico_room_assistance.plugin import RoomAssistancePlugin
+    return event.room in RoomAssistancePlugin.settings.get('rooms_with_assistance')
+
+
+def can_request_assistance_for_event(event):
+    has_room_assistance_request = event.requests.filter_by(type='room-assistance').has_rows()
```

Comment:
Are you sure you want to take withdrawn/rejected requests into account here?!

Reply:
`Request.find_latest_for_event()` might be useful here.

Reply:
`Request.find_latest_for_event` is not very useful here if I want to filter by `Request`'s state.

Reply:
well you get a single one and then check the state if it's not none. but yeah if you only care about the accepted one then querying manually is fine as well